hosts/ficolobuild/build4.nix

```diff
@@ -1,15 +1,9 @@
 # SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
 # SPDX-License-Identifier: Apache-2.0
-{
-  self,
-  lib,
-  ...
-}: {
-  imports = lib.flatten [
-    (with self.nixosModules; [
```

Comment:
I would maybe keep the `with` here just to be consistent with all the other hosts. Now if/when additional modules are added, this `with` block has to be refactored back in once reaching an arbitrary limit of say 3 modules

Reply:
Updated.